ormconfig.ts

```diff
@@ -32,17 +32,17 @@ function buildConnectionOptions(
 
   const entitiesDir =
     process.env.NODE_ENV === 'test'
```

Comment:
If we're using the same value in the `test` env and `dev/prod`, can't we just set the value directly

```suggestion
  const entitiesDir = [__dirname + '/**/*.entity.{js,ts}']
```